federationsender/inthttp/client.go

```diff
@@ -416,3 +417,35 @@ func (h *httpFederationSenderInternalAPI) LookupServerKeys(
 	}
 	return response.ServerKeys, nil
 }
+
+type eventRelationships struct {
```

Comment:
Wouldn't we usually define `EventRelationshipsRequest` and `EventRelationshipsResponse` in the `api` package?